src/features/rbac/server/projectMembersRouter.ts

```diff
@@ -91,14 +124,39 @@ export const projectMembersRouter = createTRPCRouter({
           email: input.email.toLowerCase(),
         },
       });
-      if (!user) throw new Error("User not found");
+      if (user) {
+        return await ctx.prisma.membership.create({
+          data: {
+            userId: user.id,
+            projectId: input.projectId,
+            role: input.role,
+          },
+        });
+      }
 
-      return await ctx.prisma.membership.create({
+      const invitation = await ctx.prisma.projectInvitation.create({
         data: {
-          userId: user.id,
           projectId: input.projectId,
+          email: input.email.toLowerCase(),
           role: input.role,
+          senderId: ctx.session.user.id,
+        },
+      });
+
+      const project = await ctx.prisma.project.findFirstOrThrow({
+        where: {
+          id: input.projectId,
         },
       });
+
+      if (!project) throw new Error("Project not found");
+
+      await sendProjectInvitation(
+        input.email,
+        ctx.session.user.name ?? `${process.env.EMAIL_FROM_NAME}`,
```

Comment:
Suggesting a change
- From name = EMAIL_FROM_NAME
- Title: {user.name} invited you to {project.name} -> can be done in sendProjectInvitation, you just need to pass both vars to the function